lib/glific/contacts/import.ex

```diff
@@ -13,9 +13,12 @@ defmodule Glific.Contacts.Import do
     Groups.GroupContacts,
     Partners,
     Repo,
-    Settings
+    Settings,
+    Users.User
   }
 
+  @max_concurrency System.schedulers_online() * 2
```

Comment:
Ohh double of schedulers? can we please leave it at default which is equal to System.schedulers_online() I think.

Reply:
done